### PR TITLE
start testing the wallet services

### PR DESCRIPTION
**Pull request by christophsturm**
based on #3321 This is just a very primitive test, because I don't know how to get bsq into my test wallet. but still its a start and maybe someone can help me improve it.

### CODE REVIEW

File: core/src/test/java/bisq/core/btc/wallet/BsqWalletServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.bitcoinj.wallet.Wallet;
+
+import org.junit.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class BsqWalletServiceTest {
+    @Test(expected = InsufficientBsqException.class)
+    public void throwsExceptionWhenFundsAreInsufficient() throws TransactionVerificationException, WalletException, BsqChangeBelowDustException, InsufficientBsqException {
```
Comment (blabno) [inline core/src/test/java/bisq/core/btc/wallet/BsqWalletServiceTest.java:27]: Could you separate the code with `//Given`, `//When`, `//Then` comments?

Comment (christophsturm) [reply core/src/test/java/bisq/core/btc/wallet/BsqWalletServiceTest.java:27]: I don't really like that style and I think it adds nothing. actually any comment in a test is a code smell. 
Personally I use https://github.com/dmcg/minutest for my own work, thats nice and expressive.

Comment (blabno) [reply core/src/test/java/bisq/core/btc/wallet/BsqWalletServiceTest.java:27]: Well, to me this code is not readable and tests should be very obvious. I'm not sure which part is the setup and which one is the action.

Comment (christophsturm) [reply core/src/test/java/bisq/core/btc/wallet/BsqWalletServiceTest.java:27]: yes its not very readable, and its not a very useful test, the main motivation for this pr is to make the wallet service testable at all. 

All the test code is basically "given", the last line is "when", and the "then" is in the annotation at the top (we expect a exception).

**Comment by ripcurlx** [comment]
@christophsturm Please resolve the conflict.

**Review by freimair — CHANGES_REQUESTED** [review]
please resolve the merge conflict

**Comment by christophsturm** [comment]
sorry i have no idea how to make my changes work with the changes in master.this pr was open for 6 weeks.  will just close it

**Pull request closed without merging**